index.html

```diff
@@ -0,0 +1,191 @@
+<html>
+<head> 
+  <title> </title>
```

Comment:
In my opinion, you should add a title here. It looks better.

Reply:
I suggest that you use code review language as you read in [How to give a code review](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/code-review/articles/give_code_review_basics.md) lesson.

`Remember to explain why you request a change - it might be obvious for you but not for someone else.
`

For example here, you can say `In my opinion, you should add a title here. For html best practices, it is recommended that you add a title. Also its important for search engine optimisation. [Read](https://www.w3schools.com/tags/tag_title.asp#:~:text=The%20tag%20is%20required,listing%20pages%20in%20search%20results.) for more info`